game-app/game-core/src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -25,11 +34,24 @@
 import javax.swing.JLabel;
 import javax.swing.JPanel;
 import javax.swing.JTextArea;
+import org.triplea.java.Postconditions;
 import org.triplea.java.collections.IntegerMap;
 
+/**
+ * This is the <code>JPanel</code> that shows the units to choose
+ * including the controls to make the choice.
+ *
+ * It's used by many different use cases like placing units or
+ * choosing which units take hits in a battle round.
+ */
+
 public final class UnitChooser extends JPanel {
   private static final long serialVersionUID = -4667032237550267682L;
-  private final List<ChooserEntry> entries = new ArrayList<>();
+
+  private static final Insets nullInsets = new Insets(0, 0, 0, 0);
```

Comment:
`nullInsets` sounds misleading (this is not null). Is `emptyInsets` perhaps a better name?

Reply:
Renaming to `emptyInsets` will come with the next commit.